samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -38,6 +41,11 @@ struct gnss_location_work_data {
 };
 static struct gnss_location_work_data gnss_location_work_data;
 
+
+#if defined(CONFIG_DK_LIBRARY)
+static struct k_work_delayable location_fix_led_work;
```

Comment:
`Location fix` and `acquiring location` terminology has been used in this PR. We are trying to use word `fix` only with GNSS, not with location in general. Same for `acquiring`/`acquisition`.
Depending on the context, you can use:
location, position, retrieved location, obtained location, location request, location search, location retrieval, obtaining location, requesting location, retrieving location, etc.

Reply:
OK, I see. Fixed `fix` wording for this work and to get rid of `acquire` wording for other than GNSS in this PR.